scss/atoms/_tables.scss

```diff
@@ -0,0 +1,69 @@
+// ==========================================================================
+// Typography
+// ==========================================================================
+
+table {
+	width: 100%;
+	margin-bottom: em( 20px, 16px );
+	font-size: em( 16px );
+	text-align: left;
+}
+
+thead {
+	border-bottom: 1px solid #eee;
+
+	> tr > th {
+		border-top: 0;
+	}
+}
+
+td,
+th {
+	border-top: 1px solid #eee;
+	padding: em( 12px, 16px );
+	white-space: nowrap;
+}
+
+th {
+	padding: em( 12px, 12px );
```

Comment:
It appears that way, but they aren't. The em values are different since the font size of `th` is smaller.